geckodriver.log

```diff
@@ -0,0 +1,107 @@
+1631032971180	geckodriver	INFO	Listening on 127.0.0.1:52508
```

Comment:
Note that there's  no advantage to having this checked in - different users running the tests will end up adding merge conflicts for no real value, so I suggest removing it (`git rm --cached geckodriver.log`) and adding it to the .gitignore so it doesn't return

Similarly it's best to not track the geckodriver binary - it will need to vary between operating systems, it will potentially vary for different users (e.g. if on different versions of Firefox) and it's not tiny so sharing it between multiple projects is easier - we can rely on our tests finding the driver on the PATH rather than directly in the folder